admin/app/controllers/manage_logout.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+session_start();
+
+if ($_SERVER["REQUEST_METHOD"] == "GET") {
+    // Unset all of the session variables.
+    $_SESSION = array();
+
+    // If it's desired to kill the session, also delete the session cookie.
+    // Note: This will destroy the session, and not just the session data!
+    if (ini_get("session.use_cookies")) {
+        $params = session_get_cookie_params();
+        setcookie(
+            session_name(),
+            '',
+            time() - 42000,
+            $params["path"],
+            $params["domain"],
+            $params["secure"],
+            $params["httponly"]
+        );
+    }
+
+    // Finally, destroy the session.
+    session_destroy();
+    $sessionsDeleted = 1;
+    echo $sessionsDeleted;
```

Comment:
Would we not need a check for session active here? Calling this php script at any times might be a possible threat.